backend-tests/backend-tests.hs

```diff
@@ -462,6 +464,42 @@ validateHttpResponse cid asset resp = do
 
     _ -> lift $ assertFailure $ "Could not parse header: " <> show h
 
+validateSecurityHeaders :: HasCallStack => HttpResponse -> M ()
+validateSecurityHeaders resp = do
+  validateStaticHeader resp "X-Frame-Options" "DENY"
+  validateStaticHeader resp "X-Content-Type-Options" "nosniff"
+  validateStaticHeader resp "Referrer-Policy" "same-origin"
+  validateStaticHeader resp "Permissions-Policy" $ CI.mk $ T.intercalate "," [
```

Comment:
nitpick: in the rust code you used a multiline string, might be cleaner to do the same here

https://stackoverflow.com/questions/22918837/how-can-i-write-multiline-strings-in-haskell

Reply:
Done